cdap-docs/reference-manual/source/release-notes.rst

```diff
@@ -30,6 +30,229 @@ Cask Data Application Platform Release Notes
+  container to 2048MB.
+
+- :cask-issue:`CDAP-9017` - Simplified the status, next runtime of pipelines, total number
+  of running pipelines, and drafts in the pipeline list view UI.
+
+- :cask-issue:`CDAP-9070` - Added the ability to work with Data Preparation from within
+  the CDAP Studio UI. Users can now open a Wrangler transform, wrangle the data, and save it
+  back to the CDAP Studio.
+
+- :cask-issue:`CDAP-9077` - Added a "cdap apply-pack [pack]" command to the "cdap" script
+  that allows for upgrading of individual CDAP components.
+
+- :cask-issue:`CDAP-9242` - Improved the documentation of PartitionedFileSet properties.
+
+- :cask-issue:`CDAP-9278` - Added the running of HBase coprocessor upgrades concurrently
+  on CDAP Datasets.
+
+- :cask-issue:`CDAP-9282` - Made the AppMetadataStore and ScheduleStore compatible with an
+  older data format where the app version was not present, so that it is no longer necessary
+  to update these records to the new format in the Upgrade Tool.
+
+- :cask-issue:`CDAP-9283` - The migration of ScheduleStore and AppMetadataStore from the
+  app version-less format to the new format with a version will be performed by background
+  threads running in the CDAP Master process.
+
+- :cask-issue:`CDAP-9311` - Adding support for service discovery in plugins.
+
+- :cask-issue:`CDAP-9371` - Added convenience functions to set a Date in a
+  StructuredRecord as a long or a string.
+
+- :cask-issue:`HYDRATOR-1432` - Added a timer for realtime preview to stop a run
+  automatically. By default, it is one minute.
+
+Bug Fixes
+---------
+
+- :cask-issue:`CDAP-4213` - Fixed an issue in the CDAP startup script for Windows,
+  removing the requirement of being an admin to run the CDAP process.
+
+- :cask-issue:`CDAP-5715` - Made Plugin Endpoint invocation more robust. If a plugin's
+  parent can't instantiate the plugin necessary for invoking, CDAP will attempt with other
+  parents of the plugin and try to instantiate using them before retuning error.
+
+- :cask-issue:`CDAP-6348` - Fixed a Issue in Namespace delete which caused CDAP
+  Application's test cases to fail earlier in a Windows environment.
+
+- :cask-issue:`CDAP-7138` - Fixed a problem with duplicate log lines showing for a running
+  program.
+
+- :cask-issue:`CDAP-7884` - Fixed a bug that preview cannot read from datasets in real
+  space.
+
+- :cask-issue:`CDAP-8013` - Disabled the write-to-any-dataset for sinks when in preview.
+
+- :cask-issue:`CDAP-8396` - Added a "call-to-action" when the user had successfully
+  created a new namespace.
+
+- :cask-issue:`CDAP-8668` - Modified the log viewer to only show ERROR, WARN, and INFO
+  levels of logs by default, instead of all logs as previously.
+
+- :cask-issue:`CDAP-8862` - Fix an issue with losing a few metrics when a container is
+  shutdown.
+
+- :cask-issue:`CDAP-8888` - Fixed an issue with the YARN container allocation logic so
```

Comment:
No, it's "YARN": https://hadoop.apache.org/docs/current/hadoop-yarn/hadoop-yarn-site/YARN.html